features/smoke_tests/handled.feature

```diff
@@ -19,7 +19,7 @@ Scenario: Notify caught Java exception with default configuration
     And the error payload field "events.0.exceptions.0.stacktrace" is a non-empty array
     And the event "exceptions.0.stacktrace.0.method" ends with "HandledJavaSmokeScenario.startScenario"
     And the exception "stacktrace.0.file" equals "HandledJavaSmokeScenario.java"
-    And the event "exceptions.0.stacktrace.0.lineNumber" equals 51
+    And the event "exceptions.0.stacktrace.0.lineNumber" equals 6
```

Comment:
I wonder if a comment above each step, or just at the top of each file might help people not in the R8 know understand where the line number comes from.  E.g. "See mapping.txt for R8 line number mappings"?

Reply:
Updated with a comment